### ISSUE TITLE
reply to not rendered, when only code inside a message

### ISSUE THREAD

**Issue by staabm** [kelunik/dev.kelunik.com#2]
a message like

```
:28182 

`` `
php -r 'var_dump(OPENSSL_VERSION_TEXT);'
string(25) "OpenSSL 1.0.1 14 Mar 2012"
`` `
```

is rendered like there was no reply-to at all. (I added a space between 3x backtick to make it render on github properly)

**Comment by kelunik** [comment]
Yes, and it's rendered multiple times if there is more than one `<p>`. It's easy to fix, but I have to think about it. It will probably result in:
- Prepend to first child if `firstChild.tagName === "P"`
- Prepend to message otherwise
